pkg/build/controller/strategy/custom_test.go

```diff
@@ -116,6 +116,7 @@ func mockCustomBuild() *buildapi.Build {
 						{Name: "FOO", Value: "BAR"},
 					},
 					ExposeDockerSocket: true,
+					ForcePull: true,
```

Comment:
exposedockersocket being set to true causes the pod that's created to have a mount for the dockersocket and this test is checking that all the mounts are created.  you can see that on lines 51-64, so it's necessary for the test.

you should add to this test to confirm the pull strategy is being set to Always in the pod that's created, when forcepull is set to true.

and add another test that doesn't set forcepull and confirm the pod's pull strategy is set to IfNotPresent.